takeoff_height: 1.2
landing_height: 0.4
normal_altitude: 1.0 # relative to takeoff point
